test/fixtures/awsqs-eks-cluster/index.ts

```diff
@@ -0,0 +1,487 @@
+import * as cdk from '@aws-cdk/core';
```

Comment:
optional: I wonder if it would help to include a "Generated by cdk-import" kind of comment at the top of generated files.

Reply:
Done